plugin.js

```diff
@@ -102,18 +100,15 @@ class WasmPackPlugin {
 function spawnWasmPack({
   isDebug,
   cwd,
-  withTypeScript,
-  noInstallMode
+  extraArgs
 }) {
   const bin = 'wasm-pack';
 
   const args = [
     '--verbose',
     'build',
-    '--target', 'browser',
-    ...(noInstallMode ? ['--mode', 'no-install'] : []),
     ...(isDebug ? ['--debug'] : []),
-    ...(withTypeScript ? [] : ['--no-typescript'])
+    ...(extraArgs || []),
```

Comment:
Isn't `extraArgs` alwasy an array? Since you split it above.

Reply:
You're absolutely right!